securing_apps/topics/client-registration.adoc

```diff
@@ -163,8 +163,12 @@ client = reg.create(client);
 String registrationAccessToken = client.getRegistrationAccessToken();
 ----
 
+[[_client_registration_policies]]
 === Client Registration Policies
 
+NOTE: In the future, Client Registration Policies will be removed in favour of the Client Policies described in the link:{adminguide_link}#_client_policies[{adminguide_name}].
```

Comment:
US spelling is favor

Reply:
@andymunro Thanks, this is the mistake, which I did already in the past. I am probably very used to "favour" :/ I hope to not do anymore in the future...